cloud/kubernetes/helm/yugabyte/templates/service.yaml

```diff
@@ -180,6 +180,24 @@ spec:
       containers:
       - name: "{{ .label }}"
         image: "{{ $root.Values.Image.repository }}:{{ $root.Values.Image.tag }}"
+        {{ if $root.Values.initDb}}
+        {{ if eq .name "yb-tservers" }}
+        lifecycle:
+          postStart:
+            exec:
+              command:
+                - "sh"
+                - "-c"
+                - >
+                  if ! ./bin/yb-admin -master_addresses {{ template "yugabyte.master_addresses_ps" $root }} list_tables | grep -q "postgres.pg_enum"; then
+                  if [ "$HOSTNAME" == "yb-tserver-0" ]; then
+                  while ! ./bin/yb-admin -master_addresses {{ template "yugabyte.master_addresses_ps" $root }} list_all_masters | grep -q "LEADER"; do true; done;
+                  YB_ENABLED_IN_POSTGRES=1 FLAGS_pggate_master_addresses={{ template "yugabyte.master_addresses_ps" $root }} /home/yugabyte/tserver/postgres/bin/initdb -D /tmp/yb_pg_initdb_tmp_data_dir -U postgres > /tmp/initdb.log;
+                  ./bin/yb-admin -master_addresses {{ template "yugabyte.master_addresses_ps" $root }} setup_redis_table;
```

Comment:
Are we tying `setup_redis_table` to the `initDb` step? We should not do this. Lets keep the creation of the redis table a separate step, and defaulted to false.

We want to reach a spot where we always run initDb by default and not the redis table setup.

Reply:
I think @bmatican wanted the setup redis to also be done as part of the initdb. We can remove it though if it isn't required.